pkg/codegen/python/gen.go

```diff
@@ -1909,19 +1910,16 @@ func returnTypeObject(fun *schema.Function) *schema.ObjectType {
 // Generates `def ${fn}_output(..) version lifted to work on
 // `Input`-wrapped arguments and producing an `Output`-wrapped result.
 func (mod *modContext) genFunctionOutputVersion(w io.Writer, fun *schema.Function) {
-	if !fun.NeedsOutputVersion() {
+	returnType := returnTypeObject(fun)
+
+	if returnType == nil {
+		// do not generate an output version if the function does not return an object
```

Comment:
We don't have a schema test of this? Should we add one? Could sanity check for the other languages as well. Maybe just an extension of output-funds schema

Reply:
Not having a return type is actually currently valid for promise/task-based invokes, but not for output-versioned invokes. I personally think that every invoke _must_ have a return type but we currently don't enforce it anywhere and actually allow it in sdk-gen

Reply:
Ah ok and there's already some instances in output-func schema